snapcraft/tests/test_yaml.py

```diff
@@ -346,6 +347,107 @@ def test_config_expands_filesets(self, mock_loadPlugin):
         })
 
 
+class TestYamlEnvironment(tests.TestCase):
+
+    def setUp(self):
+        super().setUp()
+        dirs.setup_dirs()
+
+        tempdirObj = tempfile.TemporaryDirectory()
+        self.addCleanup(tempdirObj.cleanup)
+        os.chdir(tempdirObj.name)
+        with open('icon.svg', 'w') as icon:
+            icon.write('<svg />')
```

Comment:
ah, yes it is because you don't have it in your example! So this can be removed.